src/client/apps/JudgeApp.ts

```diff
@@ -116,20 +120,41 @@ export class JudgeApp extends Application<JudgeAppState> {
   }
 
   submitRatingAssignment(form: RatingAssignmentForm) {
+    //Get current time
+    const end = new Date().getTime();
+    var elapsed = 0;
+    //Subtract the start time
+    //Get start time using this.state.startTime
+    if (this.state.startTime != null){
+      elapsed = end - this.state.startTime;
+      elapsed = elapsed/60000; //convert milliseconds to minutes
+    }
     this.socket.sendRequest({
       requestName: "REQUEST_SUBMIT_ASSIGNMENT",
       assignmentId: this.currentAssignment!.id,
-      ratingAssignmentForm: form
+      ratingAssignmentForm: form,
+      //submit the calculated time
+      judgetimer: elapsed
     }).then(res => {
       this.loadAssignment();
     });
   }
 
   submitRankingAssignment(form: RankingAssignmentForm) {
+    //Get current time
+    const end = new Date().getTime();
+    var elapsed = 0;
+    //Subtract the start time
+    //Get start time using this.state.startTime
+    if (this.state.startTime != null){
```

Comment:
If there is no start time this would mean elapsed would stay at 0, potentially it should set elapsed to null in a senario like this